### ISSUE TITLE
Make a real release?

### ISSUE THREAD

**Issue by manodeep** [sage-home/sage-analysis#6]
@jacobseiler I know you want to make some updates to the code quality - but once that is done, could we make a release (rather than a pre-release)?

Plus, could I please also get admin/write access to this repo :D

**Comment by jacobseiler** [comment]
Yup, I'll actually get the documentation hopefully this week and then we can set up a real release.

Added!
